service/routes.py

```diff
@@ -75,6 +75,24 @@ def get_customer(customer_id):
 
     return make_response(jsonify(customer.serialize()), status.HTTP_200_OK)
 
+######################################################################
+# DELETE A CUSTOMER
+######################################################################
+
+@app.route("/customers/<int:customer_id>", methods=["DELETE"])
+def delete_customer (customer_id):
+    """ Delete a Customer """
+    app.logger.info("Request to delete customer with id: %s", customer_id)
+    customer = Customer.find(customer_id)
+    if customer:
+        customer.delete()
+        app.logger.info("Customer with ID [%s] delete complete.", customer_id)
+    if not customer:
```

Comment:
If I remember correctly from the lectures, the delete REST API is supposed to return the status code 204 NO CONTENT even in the case when there is no customer with the id specified.

Reply:
Okay will fix and update.